lib/cadet/accounts/game_states.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Cadet.GameStates do
+  import Ecto.Repo
```

Comment:
Instead of `import Ecto.Repo` you could try using `use Cadet, :context` which includes `import Ecto.Changeset` and `alias Cadet.Repo`

If you do that you can just use `cast` instead of `Ecto.Changeset.cast` and `Repo.update!` instead of `Cadet.Repo.update!`